src/cf-forms/src/molecules/input-with-btn.less

```diff
@@ -0,0 +1,49 @@
+//
+// Input with button
+//
+
+.m-input-with-btn {
```

Comment:
Should be an `o-form__input-w-btn` since it should always be a form to allow submissions and will never have anything more than the input and button (maybe a label? We might want to discuss the lack of labels in an issue).